src/adapters/postgres/cohort-adapter.ts

```diff
@@ -9,7 +9,7 @@ import { CohortCreateDto } from "src/cohort/dto/cohort-create.dto";
 import { CohortUpdateDto } from "src/cohort/dto/cohort-update.dto";
```

Comment:
@souravbhowmik1999 https://prathamdigitalteam.atlassian.net/browse/PS-1023 please check the task. All the things should be working. Create sub task for fetching cohorts based on parent.